platform/linuxbsd/x11/detect_prime_x11.cpp

```diff
@@ -38,8 +38,9 @@
 
 #include <stdlib.h>
 
-#include <GL/gl.h>
-#include <GL/glx.h>
+#include "thirdparty/glad/glad/gl.h"
+#include "thirdparty/glx.h"
```

Comment:
It looks awkward to have both an absolute and a relative dictionary with the same name here. My first reaction was "why is `glx.h` at the root of the main `thirdparty` folder?

Any reason why this needs to be in `platform/linuxbsd/x11` instead of being in `thirdparty/glad/glad/gl.h` like `gl.h`?

Reply:
Also, `file_format.sh` finds a style issue in `glx.h`: https://github.com/godotengine/godot/actions/runs/3453104103/jobs/5763437797

Moving it to `thirdparty/glad` would solve it, otherwise you'd need to change the script like this (might be worth doing either way):
```
diff --git a/misc/scripts/file_format.sh b/misc/scripts/file_format.sh
index 1200b96ea0..408ebc2cb5 100755
--- a/misc/scripts/file_format.sh
+++ b/misc/scripts/file_format.sh
@@ -31,7 +31,7 @@ while IFS= read -rd '' f; do
         continue
     elif [[ "$f" == *"po" ]]; then
         continue
-    elif [[ "$f" == "thirdparty"* ]]; then
+    elif [[ "$f" == *"thirdparty"* ]]; then
         continue
     elif [[ "$f" == "platform/android/java/lib/src/com/google"* ]]; then
         continue
```